subprojects/docs/src/docs/release/notes.md

```diff
@@ -34,32 +36,37 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
 Gradle 6.2 ships with [dependency verification](userguide/dependency_verification.html).
 Dependency verification is a major step towards a safer ecosystem by making it possible to verify both the checksums and the signatures of dependencies and plugins used during a build.
 
-Please refer to the [userguide](userguide/dependency_verification.html) to figure out how to enable dependency verification.
+Please refer to the [user manual](userguide/dependency_verification.html) to figure out how to enable dependency verification.
 
 We would like to give special thanks to [Vladimir Sitnikov](https://github.com/vlsi) for his feedback and inspiration.
-A lot of the work on this feature is, in particular, available to previous versions of Gradle via his [Checksum Dependency Plugin](https://github.com/vlsi/vlsi-release-plugins/tree/master/plugins/checksum-dependency-plugin).
+A lot of the work on this feature is, in particular, available to previous versions of Gradle via his [Checksum Dependency Plugin](https://plugins.gradle.org/plugin/com.github.vlsi.checksum-dependency).
 
 <a name="shared-dependency-cache"></a>
 ## Shared dependency cache
 
 Improving on [relocatable dependency caches introduced in the previous release](https://docs.gradle.org/6.1.1/release-notes.html#ephemeral-ci:-reuse-gradle's-dependency-cache), Gradle 6.2 now offers the ability to **share** a dependency cache between multiple instances.
-In the context of ephemeral builds on disposable containers, this makes it possible to have a single, shared, directory between containers which contains most, if not all, the dependencies required by all builds:
 
-- each container will have access to the shared read-only dependency cache, avoiding redundant downloads between builds
-- this cache can be shared between containers without copying it, reducing the overall disk usage.
+In the context of ephemeral builds on disposable containers, this makes it possible to have a single, shared, directory between containers which contains most, if not all, the dependencies required by all builds.
```

Comment:
```suggestion
In the context of ephemeral builds on disposable containers, this makes it possible to have a single, shared directory that contains the dependencies required by all builds.
```

Reply:
Since the most if not all part is not explained in the release notes, I'm recommending that you remove it.

Reply:
done